class.jetpack-settings.php

```diff
@@ -0,0 +1,186 @@
+<?php
+
+/**
+ * All the Settings we want to sync to .com
+ *
+ * This is also used when displaying settings in the API
+ */
+class Jetpack_Settings {
+
+	public static $settings = array(
+		// also exists as "options"
+		'default_ping_status'     => 'bool',
+		'default_comment_status'  => 'bool',
+
+		// new stuff starts here
+		'blog_public'             => 'int',
+		'jetpack_sync_non_public_post_stati' => 'bool',
+		'jetpack_relatedposts_allowed' => 'bool',
+		'jetpack_relatedposts_enabled' => 'bool',
+		'jetpack_relatedposts_show_headline' => 'bool',
+		'jetpack_relatedposts_show_thumbnails' => 'bool',
+		'default_category'        => array( 'ID' =>'int', 'name' => 'string' ),
+		'post_categories'         => array( 'ID' =>'int', 'name' => 'string' ),
+		'default_post_format'     => 'string', // to check
+		'default_pingback_flag'   => 'bool',
+		'require_name_email'      => 'bool',
+		'comment_registration'    => 'bool',
+		'close_comments_for_old_posts' => 'bool' ,
+		'close_comments_days_old' => 'int' ,
+		'thread_comments'         => 'bool' ,
+		'thread_comments_depth'   => 'int' ,
```

Comment:
Not that it's a huge deal, but for things like this, should we have a way to also say that the `int` needs to be `< 10` ?

Definitely not a blocker, but in some other cases somewhere it might come up.